src/sentry/utils/tag_normalization.py

```diff
@@ -0,0 +1,89 @@
+import functools
+import re
+
+_KNOWN_TAGS = {
+    "sentry.cocoa",
+    "sentry.dart",
+    "sentry.dotnet",
+    "sentry.elixir",
+    "sentry.go",
+    "sentry.java",
+    "sentry.javascript.angular",
+    "sentry.javascript.browser",
+    "sentry.javascript.capacitor",
+    "sentry.javascript.cordova",
+    "sentry.javascript.deno",
+    "sentry.javascript.electron",
+    "sentry.javascript.ember",
+    "sentry.javascript.gatsby",
+    "sentry.javascript.miniapp",
+    "sentry.javascript.nextjs",
+    "sentry.javascript.node",
+    "sentry.javascript.node.experimental",
+    "sentry.javascript.node.hapi",
+    "sentry.javascript.raycast",
+    "sentry.javascript.react",
+    "sentry.javascript.react.expo",
+    "sentry.javascript.react.native",
+    "sentry.javascript.react.native.expo",
```

Comment:
can any of the `sentry.javascript.react.native.*` be merged?

Reply:
React and react native should definitely stay separate, as one happens purely in the browser and the other wraps browser JS, node JS, android, and iOS. As for the Expo ones, yeah, they can probably be added in with their non-Expo counterparts.

Reply:
done